corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -281,10 +310,11 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                     var choices = response.models[i].get('itemsetChoices');
                     if (choices) {
                         var $field = $($fields.get(i)),
-                            value = response.models[i].get('value');
+                            value = $field.val();
```

Comment:
Could you add a comment that you're pulling the value from the HTML because the model isn't aware of the flag for blank values?